### PR TITLE
Format list of supported resources in throttling guidance

### PR DESCRIPTION
**Pull request by FaithOmbongi**
Used tables without headers to organize the resources into unordered lists. Unfortunately, this triggers the **column-header-missing** suggestion - the price to pay.

### CODE REVIEW

File: concepts/throttling.md [hunk 1]
```diff
@@ -353,6 +284,10 @@ Other factors that affect a request cost:
   - WriteLimitExceeded - Throttling is because the write limit is exceeded.
   - ResourceUnitLimitExceeded - Throttling is because the limit for the allocated resource unit is exceeded.
 
+### Identity and access reports service limits
```
Comment (Lauragra) [inline concepts/throttling.md:287]: This section lists resources but no service limits.

File: concepts/throttling.md [hunk 2]
```diff
@@ -353,6 +284,10 @@ Other factors that affect a request cost:
...
+
+[!INCLUDE [Azure AD identity and access reports throttling documentation](../includes/throttling-aad-reports.md)]
+
 ### Information protection
```
Comment (Lauragra) [inline concepts/throttling.md:291]: The H3 here should probably include "service limits"

File: concepts/throttling.md (hunk 1 above)
Comment (FaithOmbongi) [reply concepts/throttling.md:287]: @Lauragra , the service limits were in the include file. I've moved them to this file.

File: concepts/throttling.md (hunk 2 above)
Comment (FaithOmbongi) [reply concepts/throttling.md:291]: Thanks, fixed.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**